src/index.js

```diff
@@ -277,19 +285,10 @@ class FsDatastore {
       tasks.push(pull.map(f => ({ key: this._decode(f) })))
     }
 
-    let filters = []
-
-    if (q.prefix != null) {
-      const prefix = q.prefix
-      filters.push((e, cb) => cb(null, e.key.toString().startsWith(prefix)))
-    }
-
     if (q.filters != null) {
-      filters = filters.concat(q.filters)
+      tasks = tasks.concat(q.filters.map(f => asyncFilter(f)))
```

Comment:
Please use `()` for `((f) => asyncFilter(f))`

Reply:
Why.  I've noticed it all over the code and think its just more code, *less is better*.

Do you have a publish coding standard?

Reply:
We do have contributing guidelines and linting rules (https://github.com/ipfs/community/blob/master/js-project-guidelines.md) but it is a bit outdated.

I do agree that less is more, but sometimes less means "I need to be familiar with some new ES6 thing" which turns out to be more. We kind of follow a slow transition path towards new ES features, we postpone anything that we consider ourselves that it is not as intuitive (yet) or that doesn't really add any urgent value.

Reply:
in this case there is a better way to write this actually: `tasks = tasks.concat(q.filters.map(asyncFilter))` no need to wrap in another arrow function as far as I can telll